### ISSUE TITLE
Print column_names when LQPViews are different

### ISSUE THREAD

**Issue by janetzki** [hyrise/hyrise#1740]
If we compare `LQPView`s that only differ regarding their `column_names` (e.g., `EXPECT_LQP_EQ(createViewNode1, createViewNode2)`), this difference does not become visible. Currently, the `column_names` are not printed if a comparison fails, which makes testing more difficult.
For example, `SQLTranslatorTest.CreateView` would benefit from a more verbose output.

**Comment by mrks** [comment]
Could you rephrase that? I am not sure which part you would like to change.

**Comment by janetzki** [comment]
Ok, I hope that it is clearer now.
